config/bootstrap.php

```diff
@@ -93,9 +93,9 @@
  * for a very very long time, as we don't want
  * to refresh the cache while users are doing requests.
  */
-if (!Configure::read('debug')) {
-    Configure::write('Cache._cake_model_.duration', '+1 years');
-    Configure::write('Cache._cake_core_.duration', '+1 years');
+if (Configure::read('debug')) {
+    Configure::write('Cache._cake_model_.duration', '+2 minutes');
+    Configure::write('Cache._cake_core_.duration', '+2 minutes');
```

Comment:
As this PR was mainly done because of https://github.com/cakephp/app/issues/442, I guess it fulfils his purpose of changing the default behavior of setting the cache settings.
The values are changeable at any time by the developer and worked well since cake 3 was release.